README.md

```diff
@@ -1,13 +1,24 @@
 # Accord Project Web Components
```

Comment:
May want this to include some pointer to the markdown editor or a list of components and how to install them?